src/cui.cpp

```diff
@@ -199,6 +200,11 @@ void Line::log() {
             << "\t" << recv_value << std::endl;
 }
 
+void Line::log_json() {
+  printf("{\"name\":\"%s\",\"PID\":%d,\"UID\":%d,\"RX\":%f,\"TX\":%f }",
```

Comment:
Dang in, you're absolutely right.  Processes with double quotes or other special characters that violate the json spec will screw up clients when they try to parse.  Without adding a full-fledged json lib to the project, the best (smallest) implementation (that covers everything) I could find was from here: https://github.com/nlohmann/json/blob/ec7a1d834773f9fee90d8ae908a0c9933c5646fc/src/json.hpp#L4604-L4697
Is there a better, or more preferred way for you compared to in-lining that directly into your project, or shall I go with that?!  I hate to force 90+ LOC at users for a feature most probably won't be using.  I could tuck `-j` and a json lib behind a compile time flag .. don't know just throwing ideas out there